### PR TITLE
Updated directorys - Added Port39 to list

### PR DESCRIPTION
**Pull request by techn0tr0ll**
Added Hackerspace Stralsund "Port39" to list

### CODE REVIEW

File: directory.json [hunk 1]
```diff
@@ -191,5 +191,6 @@
   "verschwoerhaus": "https://verschwoerhaus.de/feed/spaceapi",
   "vspace.one": "https://vspace.one/spaceapi.json",
   "xHain": "https://x-hain.de/spaceapi-0.13.json",
-  "z-Labor Zwickau": "http://api.service.z-labor.space/spaceapi.json"
+  "z-Labor Zwickau": "http://api.service.z-labor.space/spaceapi.json",
+  "Port39": "https://port39.de/spaceapi.json"
```
Comment (rnestler) [inline directory.json:195]: The list is sorted. Can you put it below `Pomerania Hackerspace`?

**Review by rnestler — COMMENTED** [review]
You may want to set your CORS headers correctly (see https://validator.spaceapi.io/ui/?url=https://port39.de/spaceapi.json)
`Access-Control-Allow-Origin: *`

Also you may consider supporting API v14 as well. See https://spaceapi.io/ for an example that is compatible with both v0.13 and v14

File: directory.json (hunk 1 above)
Comment (techn0tr0ll) [reply directory.json:195]: sure - i will fix this... i just checked the list and it seemed not to be sorted, sry for this

Regarding the missing header i will try my best, but until now, it's only a managed webserver... we just started our space and services so it's not the best until now

Comment (dbrgn) [reply directory.json]: Is it a static JSON file? If you use a scripting language like PHP, you can set HTTP headers: https://www.php.net/manual/en/function.header.php

Comment (techn0tr0ll) [reply directory.json]: it's just a static json file which is pushed by our door-Pi
i will check for Headers & v0.14 ^^

Comment (dbrgn) [reply directory.json]: If you can't do it for the moment, let us know, then we can merge as-is. The problem is that without the CORS headers, the endpoint will not be usable from web-based applications 🙂

Comment (techn0tr0ll) [reply directory.json]: header should just be inserted ... at least the response in dev-tools includes the "Access-Control-Allow-Origin "*"" now, if this is sufficiant?

**Comment by techn0tr0ll** [comment]
- Sorted List in directory
- Allow Origin Header set
- v14 supported (at least added differences, i could identify)

If still smth missing, pls let me know - otherwise we would be glad to get our space in the directory :)

**Comment by dbrgn** [comment]
@techn0tr0ll yes, looks great now! https://validator.spaceapi.io/ui/?url=https://port39.de/spaceapi.json Thanks for the updates.

**Review by dbrgn — APPROVED** [review]

**Pull request merged**